src/_shared/styled/StyledCardLink.tsx

```diff
@@ -0,0 +1,11 @@
+import { Link } from '@mui/material';
+import { styled } from '@mui/styles';
+
+/**
+ * A styled link to elsewhere in the app that wraps around a MUI Card element
+ * so that the entire Card area is clickable.
+ */
+export const StyledCardLink = styled(Link)(() => ({
+  textDecoration: 'none',
+  padding: '1.25 rem 0',
+})) as typeof Link;
```

Comment:
@Herraj, that's because of TypeScript's limitations on prop forwarding - the `Link` component has much more magic inside and uses the React Router's Link component rather than just an <a> HTML tag in this code.